src/pages/dataTableUtilities/getColumns.js

```diff
@@ -744,7 +744,7 @@ const COLUMNS = () => ({
     title: tableStrings.doses,
     alignText: 'right',
     sortable: true,
-    editable: true,
+    editable: false,
```

Comment:
this is weird and a bit hacky. This field is only used for focusing - I wish I called it focusable :(. Maybe I should refactor and change the field name? I dunno. But making this column not focusable - the doses gets auto-updated according to the total quantity, and if it happens to have a mismatch, I think it will be a better experience for data entry to click on the cell and edit it, only when there is a discrepancy. And saves all the focus logic drama! 🤣

Reply:
perhaps a bit hacky, but for a good reason!